lib/auth/accountrecovery.go

```diff
@@ -106,7 +92,7 @@ func (a *Server) StartAccountRecovery(ctx context.Context, req *proto.StartAccou
 // After MaxAccountRecoveryAttempts, user is temporarily locked from further attempts at recovering and also
 // locked from logging in. Modeled after existing function WithUserLock.
 func (a *Server) verifyCodeWithRecoveryLock(ctx context.Context, username string, recoveryCode []byte) error {
-	user, err := a.Services.GetUser(ctx, username, false)
+	_, err := a.Services.GetUser(ctx, username, false)
```

Comment:
I think that the method description needs updating?

Reply:
Thank you, updated!